src/Database/Schema/Dialect/MysqlDialect.php

```diff
@@ -14,14 +14,17 @@
  * @since         3.0.0
  * @license       https://opensource.org/licenses/mit-license.php MIT License
  */
-namespace Cake\Database\Schema;
+namespace Cake\Database\Schema\Dialect;
```

Comment:
Historically we have added class_alias() stubs to the old file location. Having aliases prevents breaking any external uses of these classes and give us a good spot to put deprecation warnings.